### DISCUSSION TITLE
Converting a postscript multi-page to an SVG file

### DISCUSSION THREAD

**Discussion by lpmnobre** [dlemstra/Magick.NET#1104] [Help]
Is it possible to convert a multi page postscript file to an SVG file? If so, could you please give me a snippet or a reference for doing it? Thanks so much!

**Comment by dlemstra** [comment]
I suspect that this library is not the solution that you are looking for. When Magick.NET reads a file it will change this from a vector image to a raster image and also save it as a raster image. But since SVG is also a vector format this is probably not what you want. I don't know which library has support for this though.

**Comment by lpmnobre** [comment]
Thanks so much for your prompt reply!
